internal/flags/helpers.go

```diff
@@ -58,17 +58,6 @@ func (a ByCategory) Less(i, j int) bool {
 	return iIdx < jIdx
 }
 
-func FlagCategory(flag cli.Flag, flagGroups []FlagGroup) string {
```

Comment:
I saw this function was never used and took it out. I thought it could always be re-written if it was ever needed. Also, it might be more efficient to prestore the flags one way before looking for the name rather than going over it in nested loops